client/src/components/Home.js

```diff
@@ -312,6 +332,8 @@ function Home(props) {
           <EventItem
             currentUser={props.currentUser}
             id={event._id}
+            user={user}
+            user_id={props.currentUser.id}
```

Comment:
The currentUser is already sent to the EventItem. No need to send the id, logged in status and token for the second time. Also, would be easier to just send the whole event object to the EventItem instead of sending every value one by one.